$NetBSD: patch-source4_scripting_wscript__build,v 1.2 2024/10/24 17:23:43 adam Exp $

Avoid getting strange ending # character on interpreter 
name like: 
#!/usr/pkg/bin/python2.7#

--- source4/scripting/wscript_build.orig	2024-07-29 09:03:15.670633000 +0000
+++ source4/scripting/wscript_build
@@ -12,13 +12,13 @@ if not bld.env.disable_python:
 if sbin_files:
     bld.INSTALL_FILES('${SBINDIR}',
                       sbin_files,
-                      chmod=MODE_755, python_fixup=True, flat=True)
+                      chmod=MODE_755, python_fixup=False, flat=True)
     if 'XSLTPROC_MANPAGES' in bld.env and bld.env['XSLTPROC_MANPAGES']:
         bld.MANPAGES(man_files, True)
 
 if bld.CONFIG_SET('HAVE_ADS'):
     bld.INSTALL_FILES('${BINDIR}',
                   'bin/samba-tool',
-                  chmod=MODE_755, python_fixup=True, flat=True)
+                  chmod=MODE_755, python_fixup=False, flat=True)
 
 bld.RECURSE('bin')
